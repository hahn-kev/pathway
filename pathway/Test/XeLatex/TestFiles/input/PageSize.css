@page {
	size: 8in 10in;
}

