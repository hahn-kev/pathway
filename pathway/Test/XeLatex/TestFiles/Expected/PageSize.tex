\documentclass{article} 
\usepackage{changepage}
\usepackage{float}
\usepackage{grffile}
\usepackage{graphicx}
\usepackage{amssymb}
\usepackage{fontspec}
\usepackage{fancyhdr}
\usepackage{multicol}

\begin{document} 
\thispagestyle{empty} 
\font\t="Times New Roman" at 12pt

\t{Maybe some of the LaTeX folks here can suggest how this would be integrated properly with LaTeX environments and other parameters. But it may be enough to get you started, at least. }

\end{document}
